Description: Uses 0 instead of EXIT when trapping
 Under Ubuntu, this seems harmless, but in Debian, it breaks, so I'm patching.
Author: Thomas Goirand <zigo@debian.org>
Forwarded: not-needed
Last-Update: 2013-06-11

Index: cloud-utils/bin/growpart
===================================================================
--- cloud-utils.orig/bin/growpart
+++ cloud-utils/bin/growpart
@@ -569,7 +569,7 @@ PT_UPDATE=$_RET
 debug 1 "update-partition set to $PT_UPDATE"
 
 mktemp_d && TEMP_D="${_RET}" || fail "failed to make temp dir"
-trap cleanup EXIT
+trap cleanup 0
 
 # get the ID of the first partition to determine if it's MBR or GPT
 id=$(sfdisk --id --force "${DISK}" 1 2>/dev/null) ||
